index.php

```diff
@@ -1,114 +1,30 @@
-<?php
-    // Include markdown library
-    include_once 'markdown.php';
-
-    // Load configuration
-    $config = json_decode(file_get_contents('config.json'), true);
-
-    // Get language from browser
-    $lang = substr($_SERVER['HTTP_ACCEPT_LANGUAGE'], 0, 2);
-    $suffix = '_'.$lang;
-    if ($lang == '' || $lang == $config['defaultLanguage'] || !array_key_exists($lang, $config['languages'])) {
-        $suffix = '';
-    }
-
-    // Get route
-    if (isset($_GET['uri'])) {
-        $rURI = $_GET['uri'];
-    } else {
-        $rURI = $_SERVER["REQUEST_URI"];
-    }
-
-    if ($rURI === '/') {
-        $uri = 'index'.$suffix;
-    } elseif (substr($rURI, -1) === '_')  {
-        $uri = substr($rURI, 1, -1);
-    } else {
-        $uri = substr($rURI, 1);
-    }
-
-    // Construct title
-    $title = $config['siteName'].' • '.$uri;
-
-    // Try to get markdown file
-    $markdown = file_get_contents('_pages/'.$uri.'.md');
-
-    // 404
-    if (!$markdown) {
-        header($_SERVER['SERVER_PROTOCOL'].' 404 Not Found');
-        die('Not Found');
-    }
-
-    // Compile HTML content
-    $content = Markdown($markdown);
```

Comment:
Why all this php have been removed? That really doesn't seems linked to this PR.